### ISSUE TITLE
Garbage collector

### ISSUE THREAD

**Issue by philberty** [philberty/gccpy#9]
There isn't one.

**Comment by perpetual-hydrofoil** [comment]
Challenge! Sounds difficult... current python GC is reference counting? Can you do same way?

Will this remove any issues if (i.e.) with not used, ie file handles left open?

**Comment by philberty** [comment]
Yep we can do things the same way. I am currently going for the next milestone to re-use bohem-gc as a mean time garbage collector just to have something as an option at build time possibly  its meant to be slow but might be worth while now that i am compiling more complex tests in development. I think in a few months i will start work on the garbage collector properly where then file-handles etc will all be cleaned up as you would expect in normal python.
